src/commands/wrap.ts

```diff
@@ -46,16 +43,52 @@ export const builder = (yargs: Argv): Argv =>
       description: "Use if raw directory contains wrapped files",
       type: "boolean",
       default: false
+    })
+    .option("output-file", {
+      alias: "of",
+      type: "string",
+      description: "Write output to a file. Only use when <wrapped-documents-dir> is a document",
+      conflicts: "output-dir"
+    })
+    .option("output-dir", {
+      alias: "od",
+      type: "string",
+      description: "Write output to a directory",
+      conflicts: "output-file"
     });
 
 export const handler = async (args: WrapCommand): Promise<string> => {
   try {
-    const merkleRoot = await wrap(args.rawDocumentsPath, args.wrappedDocumentsDir, {
-      schemaPath: args.schema,
-      version: args.openAttestationV3 ? "open-attestation/3.0" : "open-attestation/2.0",
-      unwrap: args.unwrap
-    });
+    const outputPathType = args.outputDir ? Output.Directory : args.outputFile ? Output.File : Output.Stdout;
+    const outputPath = args.outputDir || args.outputFile; // undefined when we use std out
+
+    if (outputPathType === Output.Stdout) {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+      // @ts-ignore
+      signale = new Signale({ logLevel: "error" });
```

Comment:
This doesn't solve the problem that we discussed
=> if I log from another file what will happen ?